components/remote_settings/src/service.rs

```diff
@@ -131,4 +152,47 @@ impl RemoteSettingsServiceInner {
         });
         active_clients
     }
+
+    fn fetch_changes(&mut self) -> Result<Changes> {
+        let mut url = self.base_url.clone();
+        url.path_segments_mut()
+            .push("buckets")
+            .push("monitor")
+            .push("collections")
+            .push("changes")
+            .push("changeset");
+        url.query_pairs_mut().append_pair("_expected", "0");
```

Comment:
You could add a comment here to note that this `0` would eventually have to be replaced by the value received in a  push notification. Using this fixed value will rely on the default CDN ttl (~1H)